### ISSUE TITLE
WarioWare Minigame Broke in 2.0.1

### ISSUE THREAD

**Issue by Essasetic** [Tarnadas/net64plus#37]
Hi the WarioWare Minigame (gamemode 8) broke in 2.0.1! When I try to host it switching game modes it just flashes text on the screen. It worked perfectly fine in 2.0 but it's basically in an unplayable state in 2.0.1! Is anybody else having this issue when trying to host a WarioWare gamemode on Net64+ or is it just me? All of the other game modes work normally but WarioWare seems to be the only one that is unplayable.

**Comment by BPopcorn** [comment]
The only thing that could cause this is faa101e4f332d7d0f2a6731a60024794f793d288.

**Comment by Tarnadas** [comment]
Yeah I changed those flags, because it prevents the invisible player "glitch" and since I heard that a workaround for fixing the instant crash on connection can be fixed by joining an empty server first, this could be also done by setting the flag like this. I'll have to play around with it and see what's the cause of WarioWare not working

**Comment by DaffyDuck54** [comment]
The Warioware Gamemode for some reason will sometimes say "Collect Star 0" but there is no star 0 in the game. Additionally the Warioware Gamemode will randomly take away all your buttons (Such as: A, B, Z, Start)
